cetmix_tower_server/models/cx_tower_server.py

```diff
@@ -762,71 +684,101 @@ def test_ssh_connection(
 
         return test_result
 
-    def _render_command(self, command, path=None):
-        """Renders command code for selected command for current server
+    def _get_connection_test_command(self):
+        """Get command used to test SSH connection
 
-        Args:
-            command (cx.tower.command): Command to render
-            path (Char): Path where to run the command.
-                Provide in case you need to override default command path
+        Returns:
+            Char: SSH command
+        """
+        command = "uname -a"
+        return command
+
+    def _get_ssh_password(self):
+        """Get ssh password
+        This function prepares and returns ssh password for the ssh connection
+        Override this function to implement own password algorithms
 
         Returns:
-            dict: rendered values
-                {
-                    "rendered_code": rendered command code,
-                    "rendered_path": rendered command path
-                }
+            Char: password ready to be used for connection parameters
         """
         self.ensure_one()
+        password = self.ssh_password
```

Comment:
should we return password with sudo? 

`self.sudo().ssh_password`

Reply:
Probably yes, thank you for noticing this!